src/main/kotlin/com/exactpro/th2/common/utils/event/MessageUtils.kt

```diff
@@ -1,23 +1,34 @@
 package com.exactpro.th2.common.utils.event
 
 import com.exactpro.th2.common.grpc.AnyMessage
+import com.exactpro.th2.common.grpc.AnyMessageOrBuilder
 import com.exactpro.th2.common.grpc.Message
 import com.exactpro.th2.common.grpc.MessageGroup
 import com.exactpro.th2.common.grpc.MessageGroupOrBuilder
 import com.exactpro.th2.common.grpc.RawMessage
 
-fun Message.toGroup(): MessageGroup = MessageGroup.newBuilder().addMessage(this).build()
-fun RawMessage.toGroup(): MessageGroup = MessageGroup.newBuilder().addMessage(this).build()
+fun Message.toGroup(): MessageGroup = MessageGroup.newBuilder().add(this).build()
+fun RawMessage.toGroup(): MessageGroup = MessageGroup.newBuilder().add(this).build()
+fun AnyMessage.toGroup(): MessageGroup = MessageGroup.newBuilder().addMessages(this).build()
 
-fun MessageGroup.Builder.addMessage(message: Message): MessageGroup.Builder = this.addMessages(AnyMessage.newBuilder().setMessage(message).build())
-fun MessageGroup.Builder.addMessage(message: RawMessage): MessageGroup.Builder = this.addMessages(AnyMessage.newBuilder().setRawMessage(message).build())
+val AnyMessageOrBuilder.sessionAlias: String
+    get() = when {
+        hasMessage() -> message.metadata.id.connectionId.sessionAlias
+        hasRawMessage() -> message.metadata.id.connectionId.sessionAlias
```

Comment:
I think we should return `null` if session alias is empty (not blank) i.e. = default value

Reply:
i think we need to use same logic for message, rawMessage, group, anymessage -> if empty return null or its gonna create misunderstanding

Reply:
> i think we need to use same logic for message, rawMessage, group, anymessage

I believe that we can use the same logic for raw, parsed and any message. But I don't think that we can apply it to a group since it can contain multiple messages. IMO it should behave the same as  `single()` method on collections or at least throw an exception if there's more than 1 session alias.

> if empty return null or its gonna create misunderstanding

Why? An empty string is a default value for unset session alias (since protobuf doesn't allow `null`s) and I think treating empty string as an absence of session alias is pretty straightforward (yeah an empty string could be also set explicitly, but we cannot determine if that's the case).